auth.css

```diff
@@ -137,11 +157,22 @@ footer a {
   color: var(--blue);
 }
 
-@media (max-width: 768px) {
+@media (max-width: 767px) {
   html {
     font-size: 8px;
   }
 
+  body {
+    margin-top: 0;
+    height: 100%;
```

Comment:
❗️ 수정요청
해당 속성때문에 브라우저 사이즈가 작아졌을 때 스크롤 영역이 충분하지 않게 잡힙니다~
아마 최소 화면 높이를 위해 추가하신 것 같으니 min-height로 변경 추천드려요~

<img width="498" alt="스크린샷 2025-03-11 오전 9 43 59" src="https://github.com/user-attachments/assets/197fdec4-88fc-4b6a-857f-ca18d1354091" />

```suggestion
    min-height: 100%;
```